etc/dbus-serialbattery/reinstall-local.sh

```diff
@@ -433,7 +433,8 @@ if [ "$can_lenght" -gt 0 ]; then
     # fastest way to check if can is installed
     if [ ! -f "/usr/lib/python3.8/site-packages/can/__init__.py" ]; then
```

Comment:
Please adapt also the check.

Reply:
Only can package was broken, the check was not affected by this PR: 

https://github.com/Louisvdw/dbus-serialbattery/pull/993/files#diff-aed43b1f692fad4f53c70146b0cb7501182ef5b1122b9cf9dc0e28d49e4223df

Here you see how it looks when installing python-can:
```
root@einstein:~# ls /usr/lib/python3.8/site-packages/can/
ls: /usr/lib/python3.8/site-packages/can/: No such file or directory

root@einstein:~# pip3 install python-can==3.3.4
Collecting python-can==3.3.4
  Using cached python-can-3.3.4.tar.gz (179 kB)
Requirement already satisfied: wrapt~=1.10 in /usr/lib/python3.8/site-packages (from python-can==3.3.4) (1.16.0)
Requirement already satisfied: aenum in /usr/lib/python3.8/site-packages (from python-can==3.3.4) (3.1.15)
Installing collected packages: python-can
    Running setup.py install for python-can ... done
Successfully installed python-can-3.3.4

root@einstein:~# ls /usr/lib/python3.8/site-packages/can/
CAN.py               __pycache__          bus.py               interface.py         io                   logger.py            notifier.py          thread_safe_bus.py   viewer.py
__init__.py          broadcastmanager.py  ctypesutil.py        interfaces           listener.py          message.py           player.py            util.py
```